iOS-TvShows/iOS-TvShows/Modules/Shows/Components/TableViewCell/ShowTableViewCell.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  ShowTableViewCell.swift
+//  iOS-TvShows
+//
+//  Created by Filip Ban on 24.11.2021..
+//
+
+import UIKit
+import Kingfisher
+import RxCocoa
+
+struct ShowTableCellItem: TableCellItem {
+    let show: Show
+    let didSelect: ((ShowTableCellItem) -> Void)?
+}
+
+extension ShowTableCellItem {
+    func cell(from tableView: UITableView, at indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(ofType: ShowTableViewCell.self, for: indexPath)
+        cell.setData(with: show)
```

Comment:
We prefer maybe a different naming, something like `func configure(with item: Show)` - in your case having `Data` in name could mislead someone that you are passing Data object (NSData)